onnxruntime/core/framework/session_state.h

```diff
@@ -515,7 +515,7 @@ class SessionState {
   // prepacked_weights_container_ can be nullptr if no caching is required for prepacked weights
   PrepackedWeightsContainer* const prepacked_weights_container_{};
 
-#ifdef ENABLE_TRAINING
+#ifdef ENABLE_TRAINING_CORE
 #ifndef DISABLE_ABSEIL
   InlinedHashMap<InlinedVector<int>, InlinedHashSet<NodeIndex>> to_be_executed_nodes_;
```

Comment:
this is for orttrainer. maybe we can put a comment here, to remove it when remove orttrainer.